plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -277,9 +277,9 @@ public final void testBundleExecutionEnvironment() throws IOException {
   @Test
   public final void testGuavaImportVersions() throws IOException {
     checkDependencyDirectives(
```

Comment:
We should have a similar test for `com.google.cloud.tools.appengine`

Reply:
I'm not sure this test has actually proved useful. I don't think it's caught anything yet.